Content.Shared/Medical/DefibrillatorComponent.cs

```diff
@@ -46,6 +33,18 @@ public sealed partial class DefibrillatorComponent : Component
     [DataField("writheDuration"), ViewVariables(VVAccess.ReadWrite)]
     public TimeSpan WritheDuration = TimeSpan.FromSeconds(3);
 
+    /// <summary>
+    ///     ID of the cooldown use delay.
+    /// </summary>
+    [DataField]
+    public string DelayId = "defib-delay";
+
+    /// <summary>
+    ///     Cooldown after using the defibrillator.
+    /// </summary>
+    [DataField("zapDelay")]
```

Comment:
```suggestion
    [DataField]
```

Reply:
I'm not kidding this is one of the most embarrassing mistakes I've ever made